packages/xdl/src/EmbeddedAssets.ts

```diff
@@ -265,6 +297,9 @@ async function _maybeConfigureExpoUpdatesEmbeddedAssetsAsync(config: EmbeddedAss
   if (fs.existsSync(path.join(supportingDirectory, 'Expo.plist'))) {
```

Comment:
should we warn if the user has `expo-updates` installed but we can't find `Expo.plist`?